scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -712,11 +727,25 @@ export function AuthoringDirective(
                 return false;
             }
 
-            $scope.showCustomButtons = function(item) {
-                if ($location.path() === '/workspace/personal') {
-                    return false;
-                }
-                return item.task && item.task.desk && item.state !== 'draft' || $scope.dirty;
+            // ToDesk -- Send an Item to a desk
+
+            // closeAndContinue -- Create an update of an item and Close the item.
+
+            // publish -- publish an item
+
+            // publishAndContinue -- Publish an item and Create an update.
+
+            $scope.showCustomButtons = () => {
```

Comment:
I don't like this function which seems to be a check if any button is visible also setting those other variables, shouldn't that be rather done when item is updated and this would only return the status/